libmscore/fermata.cpp

```diff
@@ -113,6 +113,19 @@ void Fermata::write(XmlWriter& xml) const
       xml.etag();
       }
 
+//---------------------------------------------------------
+//   subtype
+//---------------------------------------------------------
+
+int Fermata::subtype() const
```

Comment:
I would call it `normalizedName`. What do you think? I spent few minutes looking at the method to get that this just returns `"SymName"Above` in both cases `"SymName"Below` and `"SymName"Above`. Also, related comment is worth adding in the method description.

The same for `Articulation::subtype()`.

Reply:
My thinking was that `subtype()` should return the same value regardless of direction. It is a virtual method inherited from the Element class.

Reply:
Okay, I'll create related issue to rename it, so.

Reply:
Are you thinking of renaming Element::subtype()? I do not think that is necesssary. Perhaps Element::subtype() could return an enum value, which would be determined by the symId. That would mean Articulation::subtype() would be quite lengthy, but it would be doable.